test/unit/sagas/projects.js

```diff
@@ -226,8 +228,7 @@ test('updateProjectSource', (assert) => {
     updateProjectSource(scenario.projectKey, 'css', 'p {}'),
   ).
     next().select().
-    next(scenario.state).call(saveCurrentProject, scenario.state).
-    next().isDone();
+    next(scenario.state).isDone();
```

Comment:
So it seems like these tests aren’t really making as meaningful assertions about saga behavior as they were before—I’m guessing because we have now internalized the check for `isPristineProject` which, at a guess, is not the case in the “scenario” that’s been set up here? Assuming that guess is correct, should be pretty straightforward to fix in the scenario file.

I assume this is what you meant by “trouble creating tests for the projects saga”?